test/IECoreHoudini/SceneCacheTest.py

```diff
@@ -33,40 +33,56 @@
 ##########################################################################
 
 import os
+import uuid
 import hou
 import IECore
 import IECoreHoudini
 import unittest
 
 class TestSceneCache( IECoreHoudini.TestCase ) :
 	
-	__testFile = "test/test.scc"
-	__testOutFile = "test/testOut.scc"
-	__testLinkedOutFile = "test/testOut.lscc"
-	__testHip = "test/test.hip"
-	__testBgeo = "test/test.bgeo"
-	__testBgeoGz = "test/test.bgeo.gz"
-	__testGeo = "test/test.geo"
+	if hou.applicationVersion() >= 16:
+		POINT_POSITION_ATTRIBS = ['P']
+	else:
+		POINT_POSITION_ATTRIBS = ['P', 'Pw']
+		
+	def setUp(self):
+		hou.hipFile.clear()
+		
+		self._testFile = "test/test{TEST_NAME}.scc".format(TEST_NAME=self.id())
```

Comment:
This is because it was possible for one test to write to test/testOut.scc, and then die before removing the file, which would cause other tests to fail really unpredictably. This gives them unique names so that the tests don't influence each other.